components/action-mgt/org.wso2.carbon.identity.action.execution/src/test/java/org/wso2/carbon/identity/action/execution/util/ActionExecutorConfigTest.java

```diff
@@ -302,4 +303,58 @@ public void testGetExcludedParamsInActionRequestForEmptyConfigForAllAndDefinedTy
                 actionExecutorConfig.getExcludedParamsInActionRequestForActionType(ActionType.PRE_ISSUE_ACCESS_TOKEN);
         assertEquals(excludedHeaders, Collections.emptySet());
     }
+
+    @Test
+    public void testGetHttpReadTimeoutInMillis() {
+
+        Map<String, Object> configMap = new HashMap<>();
+        configMap.put("Actions.HTTPClient.HTTPReadTimeout", "5000");
+        when(mockIdentityConfigParser.getConfiguration()).thenReturn(configMap);
+        Assert.assertEquals(5000, actionExecutorConfig.getHttpReadTimeoutInMillis());
+    }
+
+    @Test
+    public void testGetHttpReadTimeoutInMillisWithoutNumberFormat() {
```

Comment:
Let's rename the method to denote it's an invalid config.
Like 'testGetHttpReadTimeoutInMillisForInvalidConfig'

Reply:
Updated in https://github.com/wso2/carbon-identity-framework/pull/5935/commits/edba811642bfb3eb64866ebfef8efd9d67ae484e